### ISSUE TITLE
Create a new release

### ISSUE THREAD

**Issue by iDuuck** [Soundnode/soundnode-app#638]

**Comment by Pitros** [comment]
What?

**Comment by weblancaster** [comment]
What do you mean? you want a new release out @iDuuck ?

**Comment by cenderin** [comment]
What an interesting "Issue"...

**Comment by iDuuck** [comment]
Yeah, there are so many things changed. There should be updates on the app more regularly.

**Comment by weblancaster** [comment]
I agree with you @iDuuck we should release more often (bi-weekly based release), the problem is that at the moment we don't have a better system to release new versions, what I mean by that is that the user will need to go to the website and manually download the new release, this can be tedious and people already complained about it.

also, this spam time gives me time to test what is implemented in master, as example right now we have a bug that was introduced by a PR which seemed alright at the time I tested but as I used the app more naturally I noticed the bug.

I will try release a new version by next Monday..thanks.

**Comment by cenderin** [comment]
@weblancaster what kind bug? Maybe post a issue with more details and maybe some of us can debug with you.

**Comment by weblancaster** [comment]
Yeah I should have created before.. but here it is #639

**Comment by calvwng** [comment]
I think @iDuuck may have wanted to see the roadmap for what changes would be deployed in upcoming releases? If that's the case, I think the Waffle kanban board is the closest thing.
https://waffle.io/Soundnode/soundnode-app

**Comment by JonasGroeger** [comment]
> the problem is that at the moment we don't have a better system to release new versions, what I mean by that is that the user will need to go to the website and manually download the new release

See #777 and #780 and #775

**Comment by JonasGroeger** [comment]
For Debian, Ubuntu, Elementary and Mint you can use

    curl -s https://packagecloud.io/install/repositories/JonasGroeger/soundnode/script.deb.sh | sudo bash
    sudo apt-get install soundnode

This is automatically updated whenever a new release is available.
